com.vivaldi.Vivaldi.yaml

```diff
@@ -0,0 +1,124 @@
+app-id: com.vivaldi.Vivaldi
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.chromium.Chromium.BaseApp
+base-version: '22.08'
+command: vivaldi
+separate-locales: false
+finish-args:
+  - --device=all # Webcams and others
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf # For GNOME proxy resolution
+  - --env=GIO_EXTRA_MODULES=/app/lib/gio/modules # For GNOME proxy resolution
+  - --env=GSETTINGS_BACKEND=dconf # For GNOME proxy resolution
+  - --env=GTK_PATH=/app/lib/gtkmodules
+  - --filesystem=/run/.heim_org.h5l.kcm-socket
+  - --filesystem=home/.local/share/applications:create # Allow for PWAs
+  - --filesystem=home/.local/share/icons:create # Allow for PWAs
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-download # Default download directory
+  - --filesystem=xdg-run/dconf # For GNOME proxy resolution
+  - --filesystem=xdg-run/pipewire-0
+  - --filesystem=~/.config/dconf:ro # For GNOME proxy resolution
+  - --filesystem=~/.config/kioslaverc # For KDE proxy resolution (KDE5 only)
+  - --persist=.local/lib # Extra FFMpeg saved here
```

Comment:
This does not seem to work from the log @TheEvilSkeleton posted?

Reply:
Yeah I switched it back.